### ISSUE TITLE
Updates of big wire net cause tps reduce

### ISSUE THREAD

**Issue by hohserg1** [asiekierka/CharsetMC#460]
How to represent:
1) Check tps
2) Place red wire matrix, at least 2*2 chunks
3) Connect to clock generator
clock generator will like frozen after connection
4) Check tps

Mods:
SimpleLogic-Wires-0.5.6.3
SimpleLogic-Gates-0.5.6.1
MCMultiPart-2.5.3
Charset-Lib-0.5.6.6

Vedeo:
https://youtu.be/Ft8YQFzF51U
modpack at video have some other mods, but I got same effect with clear modpack

The reason why I don't think it should be: I tried same setup with ProjectRed and it work without lag

**Comment by asiekierka** [comment]
Yeah, SimpleLogic is not terribly performant - due to low interest in the mod and having moved on to working on Fabric later, I never really got to the "optimization" stage of development. I don't think I have it in me to do a rewrite, unfortunately.

**Comment by hohserg1** [comment]
Uh, sadly
